webapp/src/Service/ScoreboardService.php

```diff
@@ -412,17 +433,34 @@ public function calculateScoreRow(Contest $contest, Team $team, Problem $problem
             //   - either it's still ongoing (pending judgement, could be correct)
             //   - or already judged to be correct (if it's judged but != correct, it's not a first to solve)
             // - or the submission is still queued for judgement (judgehost is NULL).
-            $firstToSolve = 0 == $this->em->getConnection()->fetchColumn('
+            if ($useExternalJudgements) {
```

Comment:
I think this would be more readable by just repeating the SQL copy instead of avoiding repetition in an complex way.
Feel free to leave it like it is now since it is a matter of taste.

Reply:
After trying it I have to agree so I will do it